app/docs/enterprise/0.32-x/getting-started/quickstart.md

```diff
@@ -76,7 +76,7 @@ $ kong reload
 
 Now that you have Kong EE running you can interact with the Admin API and GUI.
 
-To begin, go to [Adding your API &rsaquo;][adding-your-api]
+To begin, go to [Configuring a Service &rsaquo;][configuring-a-service]
```

Comment:
you will also need to change the link at the bottom of this document in order for the link to work.